### ISSUE TITLE
[GS v4] head table diff

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans#221]
Review diff of head tables between David Berlow's builds @ https://github.com/googlefonts/GoogleSans-fb/commit/a8d2737ba302a95dcb692c8b8b05bdf481631cb0 and our v3.003 production releases

**Comment by chrissimpkins** [comment]
Roman var font

- ymax decreased to 1000 from 1056

<details>
<summary> diff</summary>

```diff
--- A/GoogleSans[opsz,wght].ttf	2020-09-16T16:50:43-04:00
+++ B/GoogleSansGrades-VF.ttf	2021-02-25T11:12:08.527802-05:00
@@ -5,16 +5,16 @@
     <!-- Most of this table will be recalculated by the compiler -->
     <tableVersion value="1.0"/>
     <fontRevision value="3.003"/>
-    <checkSumAdjustment value="0x93fc0fef"/>
+    <checkSumAdjustment value="0x9552e56"/>
     <magicNumber value="0x5f0f3cf5"/>
     <flags value="00000000 00000011"/>
     <unitsPerEm value="1000"/>
-    <created value="Thu Jul  6 09:41:44 2017"/>
-    <modified value="Wed Sep 16 20:50:43 2020"/>
+    <created value="Thu Jul  6 05:41:44 2017"/>
+    <modified value="Thu Feb 25 16:12:08 2021"/>
     <xMin value="-354"/>
     <yMin value="-381"/>
     <xMax value="3284"/>
-    <yMax value="1056"/>
+    <yMax value="1000"/>
     <macStyle value="00000000 00000000"/>
     <lowestRecPPEM value="6"/>
     <fontDirectionHint value="2"/>
```

</details>

**Comment by chrissimpkins** [comment]
Italic var font

- ymax decreased to 1000 from 1056

<details>
<summary> </summary>

```diff
--- A/GoogleSans-Italic[opsz,wght].ttf	2020-09-16T16:50:43-04:00
+++ B/GoogleSansItalicGrades-VF.ttf	2021-02-25T11:12:48.431719-05:00
@@ -4,17 +4,17 @@
   <head>
     <!-- Most of this table will be recalculated by the compiler -->
     <tableVersion value="1.0"/>
-    <fontRevision value="3.003"/>
-    <checkSumAdjustment value="0x4cb089a0"/>
+    <fontRevision value="3.001"/>
+    <checkSumAdjustment value="0x584e2258"/>
     <magicNumber value="0x5f0f3cf5"/>
     <flags value="00000000 00000011"/>
     <unitsPerEm value="1000"/>
-    <created value="Tue May  2 10:18:26 2017"/>
-    <modified value="Wed Sep 16 20:50:43 2020"/>
+    <created value="Tue May  2 06:18:26 2017"/>
+    <modified value="Thu Feb 25 16:12:48 2021"/>
     <xMin value="-314"/>
     <yMin value="-381"/>
     <xMax value="3284"/>
-    <yMax value="1056"/>
+    <yMax value="1000"/>
     <macStyle value="00000000 00000010"/>
     <lowestRecPPEM value="6"/>
     <fontDirectionHint value="2"/>
```

</details>

**Comment by chrissimpkins** [comment]
Likely due to https://github.com/googlefonts/GoogleSans-fb/issues/29 and needs to be addressed. U+FFFD must keep yMin and yMax points.

**Comment by chrissimpkins** [comment]
> Likely due to googlefonts/GoogleSans-fb#29 and needs to be addressed. U+FFFD must keep yMin and yMax points.

Confirmed.

Fixed downstream in https://github.com/googlefonts/GoogleSans-fb/commit/11a9952da1983fca2ec3ae09c8a2589a2edbb0a5 and https://github.com/googlefonts/GoogleSans-fb/commit/a43ca6ef06bef655d1641afac450d6829f8cb0a1

**Comment by chrissimpkins** [comment]
Confirmed that there is no longer a regression now that U+FFFD changes were reverted and upstream shaping was replaced. This will be closed by #240
